browser.js

```diff
@@ -240,7 +240,11 @@ function init() {
 
 	// link the theme if it was changed while the app was closed
 	if (process.platform === 'darwin') {
-		storage.set('darkMode', osxAppearance.isDarkMode());
+		const isDarkMode = osxAppearance.isDarkMode();
```

Comment:
`osxAppearance.isDarkMode()` always returns `true` for me, making this entire block just set `darkMode` is the storage to `true` all the time. Which makes me wonder what this block accomplishes.